lib/widgets/content.dart

```diff
@@ -22,7 +23,24 @@ class MessageContent extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BlockContentList(nodes: content.nodes);
+    return InheritedMessage(message: message, child: BlockContentList(nodes: content.nodes));
```

Comment:
nit:
```suggestion
    return InheritedMessage(message: message,
      child: BlockContentList(nodes: content.nodes));
```

That way the main payload (the BlockContentList) is kept prominently visible.